recipe/meta.yaml

```diff
@@ -271,20 +276,33 @@ outputs:
         # decide on the cuda variant
         # https://github.com/conda-forge/tensorflow-feedstock/issues/162
         - {{ compiler('cuda') }}                 # [cuda_compiler_version != "None"]
-        - bazel 4.2.2
+        - bazel >=5.2.0
+        - libabseil
+        - grpc-cpp
+        - numpy 1.20.*  # [py<310]
+        - numpy 1.21.*  # [py>=310]
+        - openssl
+        - libprotobuf
+        - {{ pin_subpackage('tensorflow-base', exact=True) }}
```

Comment:
It seems to me that all this should suffice as a host-dep (except perhaps in cross-compilation) - why do we need it in build as well?

Reply:
Why is this needed in the build requirements anyway?

Reply:
I don't see `tensorflow-base` as a build requirement in original recipe.

We do something similar in `pyside2`, but that is really conditioned on cross compilation.
https://github.com/conda-forge/pyside2-feedstock/blob/main/recipe/meta.yaml#L32

I don't think this `tensorflow-base` should be in the `build` if we are not cross compiling. And maybe not in general.